__tests__/dashboard.test.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
```

Comment:
could we group all type of tests (UT, IT, FRT) into `test` folder like `src` folder

Reply:
Does the __tests__ folder count?